.ci/openshift-ci/README.md

```diff
@@ -0,0 +1,3 @@
+# OpenShift CI files
+
+This directory contain scripts and files used by the openshift-ci operator to test kata. For further details read this [README](https://github.com/kata-containers/tests/tree/master/.ci/openshift-ci/README.md) file.
```

Comment:
Nit: very long line.

Reply:
I deleted the `README.md` so not need to this. :)